docs/android/emulator/device-manager.md

```diff
@@ -0,0 +1,426 @@
+---
+title: "Managing Virtual Devices with the Android Device Manager"
+description: "This article explains how to use the Android Device Manager to create and configure Android Virtual Devices (AVDs) that emulate physical Android devices, for .NET MAUI. You can use these virtual devices to run and test your app without having to rely on a physical device"
+ms.date: 11/18/2021
+no-loc: ["user.config"]
+---
+
+# Managing virtual devices with the Android Device Manager
+
+This article explains how to use the Android Device Manager to create and configure Android Virtual Devices (AVDs) that emulate physical Android devices. You can use these virtual devices to run and test your app without having to rely on a physical device.
+
+> [!IMPORTANT]
+> Enable hardware acceleration for the android devices. For more information, see [Hardware Acceleration for Emulator Performance](hardware-acceleration.md).
+
+<!--
+::: zone pivot="windows"
+-->
+
+## Android Device Manager on Windows
+
+You use the Android Device Manager to create and configure an Android Virtual Devices (AVD) that run in the [Android Emulator](debug-on-emulator.md). Each AVD is an emulator configuration that simulates a physical Android device. This makes it possible to run and test your app in a variety of configurations that simulate different physical Android devices.
+
+:::image type="content" source="media/device-manager/win/01-devices-dialog-sml.png" alt-text="Screenshot of the Android Device Manager in the Devices tab." lightbox="media/device-manager/win/01-devices-dialog.png":::
+
+## Requirements
+
+To use the Android Device Manager, you'll need the following items:
+
+- Visual Studio 2022 version 17.1 Preview 3. Visual Studio Community, Professional, and Enterprise editions are supported.
+
+- The **Android SDK API Level 30** or later. Be sure to install the Android SDK at its default location if it isn't already installed: _C:\\Program Files (x86)\\Android\\android-sdk_.
+
+- The following packages must be installed in the:
+
+  - **Android SDK Tools 5.0** or later
+  - **Android SDK Platform-Tools 31.0.3** or later
+  - **Android SDK Build-Tools 30.0.2** or later
+  - **Android Emulator 30.8.4** or later
+
+  These packages should be displayed with **Installed** status as seen in the following screenshot:
+
+  :::image type="content" source="media/device-manager/win/02-sdk-tools.png" alt-text="Installing Android SDK Tools.":::
+
+When you install the **Mobile development with .NET** workload in Visual Studio, and select the optional **.NET MAUI (Preview)** workload, everything is installed for you. For more information on setting up .NET MAUI with Visual Studio, see [Installation](../../get-started/installation.md).
```

Comment:
I think you can just say that "When you install blah workload..., everything is installed for you". - no need to mention the optional workload. It's checked by default and this will save having to return to this doc to edit it out at GA.